src/php/Compiler/Analyzer/TypeAnalyzer/SpecialForm/InvokeSymbol.php

```diff
@@ -27,6 +28,12 @@ public function analyze(PersistentListInterface $list, NodeEnvironmentInterface
             $env->withContext(NodeEnvironmentInterface::CONTEXT_EXPRESSION)->withDisallowRecurFrame()
         );
 
+        /*var_dump(gettype($f));
+        var_dump($this->isInline($f, count($list) - 1));*/
```

Comment:
Missing comments 👀

Reply:
Fixed. :sweat:

Reply:
Btw instead of `var_dump()` try using `dump()` 😉